postcommit.py

```diff
@@ -0,0 +1,642 @@
+#			shows number of added and deleted lines in decimal notation for each files. 
+#
+#			@param : changeset
+#			
+#			@return : {'Removed': [], 'Added': [], 'Modified': [{'AddedLines': '1', 'File': 'test', 
+#				'RemovedLines':'2'}]}         						
+#	*************************************************************************************************************** 
+    	def filestatus(self, changeset):
+        	cmd = "git show --name-status --pretty='format:CMHash:%H' " + changeset
+        	commit_info = status_check(cmd, self.module_dir)
+        	commit_sts_pattern = re.compile('CMHash:([a-z0-9]+)\n+(([AMD]\s+(.*?)\n)*)?')
+        	commit_sts_detail = commit_sts_pattern.findall(str(commit_info[0]))
+
+        	result = {'Modified':[], 'Added': [], 'Removed':[]}
+
+		cmd = "git show --numstat --pretty='format:CMHash:%H' " + changeset
+		filter_info = status_check(cmd,self.module_dir)
+		filter_sts_pattern = re.compile('CMHash:([a-z0-9]+)\n+(([0-9\-]+\s+[0-9\-]+\s+(.*?)\n)*)?')
+		filter_sts_detail = filter_sts_pattern.findall(str(filter_info[0]))
+
+		if commit_sts_detail:
+           		commit_sts_detail = commit_sts_detail[0]
+			filter_sts_detail = filter_sts_detail[0] 
+
+		if commit_sts_detail:
+            		if (commit_sts_detail[0] == changeset and commit_sts_detail[1] != ''):
+               			changes = commit_sts_detail[1].strip('\n')
+				filter_change = filter_sts_detail[1].strip('\n')
+				filter_change = filter_change.split('\n')
+				i = 0
+                		for detail in changes.split('\n'):
+                    			detail = detail.split('\t')
+                   			status = detail[0]
+                    			filename = escapeChars(detail[1])
+
+					insertion_pattern = re.compile('([0-9\-]+)\t([0-9\-]+)')
+
+					file_numstats = { 'File':[],'AddedLines':[],'RemovedLines':[] }
+
+					num_stat = insertion_pattern.findall(filter_change[i])
+					i = i+1
+					file_numstats['File']=filename
+					if num_stat:
+						num_stat = num_stat[0]
+						file_numstats['AddedLines']=num_stat[0]
+						file_numstats['RemovedLines']=num_stat[1]
+					else:
+						file_numstats['AddedLines']=0
+						file_numstats['RemovedLines']=0
+
+                    			if status == 'A':
+                      				result['Added'].append(file_numstats)
+                    			elif status == 'M':
+                        			result['Modified'].append(file_numstats)
+                    			elif status == 'D':
+                        			result['Removed'].append(file_numstats)
+
+        	return result
+
+	
+#       ***************************************************************************************************************
+#	parseCommits :
+#
+#			Find commits and newchangesets details
+#
+#			@param : commitlog,branch
+#			
+#			@return : None
+#       ***************************************************************************************************************
+    	def parseCommits(self, commitlog, branch):
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename method "parseCommits" to match the regular expression ^[a-z_][a-z0-9_]{2,}$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=python%3AS100)